e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/bss-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,77 @@
+import relative from '../../../relativeURL';
+import facilityPage from '../../../pages/facilityPage';
+import amendmentsPage from '../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../fixtures/deal-AIN';
+import dateConstants from '../../../../../../e2e-fixtures/dateConstants';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1 } from '../../../../../../e2e-fixtures';
+
```

Comment:
Facility End date only occurs on GEF deals :)

Reply:
I've renamed this file slightly as a personal preference

Reply:
Thanks @AlexBramhill , finding this a little odd though - currently we have this structure:

```
facility-end-date/bss-deal-expect-no-facility-end-date-features.spec.js
facility-end-date/feature-flag-disabled/*.spec.js
facility-end-date/feature-flag-enabled/*.spec.js
```

Seems it would make more sense to have `bss-deal-expect-no-facility-end-date-features.spec.js` in the `feature-flag-disabled` directory - or maybe the tests are already covred in the "disabled" directory?

Reply:
I've set it up like this so that this test is run _both_ when the facility end date feature flag is enabled _and_ when it is disabled. In the case of a non-GEF facility we should check it doesn't have any facility end date features both when the FF is enabled _and_ when it is disabled.

Reply:
OK thanks @BethThomas141 makes sense, agree it's cleaner not to duplicate.

I think if we had more DRY cypress commands we could have the BSS/EWCS test in both of the feature flag directories which IMO would be clearer to someone without context - I wouldn't have known it's run twice with the feature flag disabled and enabled without these PR comments